package.json

```diff
@@ -103,10 +101,12 @@
     "apollo-server-express": "^2.24.1",
     "apollo-server-express-v3": "npm:apollo-server-express@3.6.2",
     "async": "^3.2.2",
+    "semver": "7.5.4",
```

Comment:
FYI: I had to add the semver version here because NPM was complaining because a subdependency relied on an older semver version which was then stored under `node_modules/semver` (main semver version) and this version was vulnerable. `npm audit` failed.